src/diart/blocks/aggregation.py

```diff
@@ -130,10 +132,12 @@ def __init__(
         step: float,
         latency: Optional[float] = None,
         strategy: Literal["mean", "hamming", "first"] = "hamming",
+        cropping_mode: Literal["strict", "loose", "center"] = "loose"
```

Comment:
There is also a lot of cropping going on in `AggregationStrategy` subclasses, we should add this parameter there as well so it remains consistent across all `DelayedAggregation` operations.

Reply:
Solved with e7d6dd7

Reply:
Attempt to solve this in 4866586. Do you have a quick way of testing that?

Reply:
Yes I'll test it and get back to you, thanks!